### PR TITLE
Gets rid of the spacing changes for future reviewers

### PR DESCRIPTION
**Pull request by lamagnifica**
@catong 
Also light editing (learner for student in most places, select for click in most places, section for chapter)

### CODE REVIEW

**Comment by lamagnifica** [comment]
@srpearce if you have time this is another of those spacing-related + light style guide edit PRs....

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 1]
```diff
@@ -35,11 +36,11 @@ the number of attempts made, and the date and time of the submission.
 View One Student's Submission History
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "Student's" -> "Learner's"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 2]
```diff
@@ -60,12 +61,12 @@ the viewer.
 
 .. _Interpret a Student Submission History:
 
-============================================================
+==========================================
 Interpret a Student's Submission History
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "Student's" -> "Learner's"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 3]
```diff
@@ -207,11 +208,11 @@ reported ``Score`` and the values for ``student_answers``, ``attempts``, and
 Report All Students' Answer Submissions
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "Students'" -> "Learners'"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 4]
```diff
@@ -207,11 +208,11 @@ reported ``Score`` and the values for ``student_answers``, ``attempts``, and
...
 ============================================================
 
-Before you can download a report of all student answers for a problem, you need
+Before you can download a report of all learner answers for a problem, you need
 the :ref:`unique identifier<find_URL>` of the problem that you want to
 investigate.
 
-To download a report of the answers submitted for a problem by every student,
+To download a report of the answers submitted for a problem by every learner,
 called the Student State report, follow these steps.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: Reads a bit as if "called the Student State report" modifies "every learner". Possibly rewrite as: "To download the Student State report, which is a report of all learner answers for a problem, you need..."

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 5]
```diff
@@ -157,15 +158,15 @@ after the student submitted the answer. This record includes
 
 **Record 4: Problem Retried (Browser)**
 
-When a problem gives students multiple attempts at the right answer, and the
-student tries again, an additional record is added when a student selects
+When a problem gives learners multiple attempts at the right answer, and the
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: Suggest: "right answer" -> "correct answer"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 6]
```diff
@@ -304,7 +305,7 @@ Student Answer Distribution
 ****************************************
 
 For certain types of problems in your course, you can download a CSV file with
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 7]
```diff
@@ -315,11 +316,12 @@ data is included in the file for problems of these types.
...
+learners. For example, for a problem that has a total of five possible answers
 the file includes up to five rows, one for each answer selected by at least one
-student. For problems with **Randomization** enabled in Studio (sometimes
+learner. For problems with **Randomization** enabled in Studio (sometimes
 called rerandomization), there is one row for each problem-variant-answer
-combination selected by your students. See :ref:`Problem Settings`.
+combination selected by your learners. For more information, see :ref:`Problem
+Settings`.
 
 The CSV file contains the following columns.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 8]
```diff
@@ -375,10 +377,10 @@ Download the Student Answer Distribution Report
 ===================================================
 
 An automated process runs periodically on the edX servers to update the CSV
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 9]
```diff
@@ -375,10 +377,10 @@ Download the Student Answer Distribution Report
...
-file of student answer data. A link to the most recently updated version of the
+file of learner answer data. A link to the most recently updated version of the
 CSV file is available on the Instructor Dashboard.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 10]
```diff
@@ -445,7 +447,7 @@ answers were given before and after you made the change.
 report in a spreadsheet?**
 
 Some spreadsheet applications can alter the data in the CSV report for display
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 11]
```diff
@@ -445,7 +447,7 @@ answers were given before and after you made the change.
...
-purposes. For example, for different student answers of "0.5" and ".5" Excel
+purposes. For example, for different learner answers of "0.5" and ".5" Excel
 correctly includes the two different lines from the CSV, but displays the
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 12]
```diff
@@ -473,11 +475,11 @@ If you notice characters that do not display as expected in a spreadsheet, try
 a different spreadsheet application such as LibreOffice or Apache OpenOffice to
 open the CSV file.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_student.rst [hunk 13]
```diff
@@ -145,28 +145,28 @@ To download learner data, follow these steps.
 #. To download data about enrolled learners in a CSV file, select **Download
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_student.rst]: "CSV" -> ".csv"

File: en_us/shared/building_and_running_chapters/running_course/course_staffing.rst [hunk 14]
```diff
@@ -32,11 +41,13 @@ Staff role or the Admin role.
...
-* View the course before the Course Start Date. 
+* View the course before the Course Start Date.
 
-* Enroll and unenroll students.
+* Enroll and unenroll learners.
 
-* Access student grades.
+* Access learner grades.
+
+* Reset learner attempts to answer a question correctly.
```
Comment (catong) [inline en_us/shared/building_and_running_chapters/running_course/course_staffing.rst]: "Reset learner attempts to answer..." was correctly removed from the "Staff Role" section and moved down to the "Admin Role" section by an OSPR correcting this doc bug. (So it should not have been restored here). 
I can't comment on it below (line 60) but there, "student" should be "learner".

**Comment by catong** [comment]
@lamagnifica A few comments, esp. see https://github.com/edx/edx-documentation/pull/673#discussion_r44947763 otherwise :+1:.

**Pull request merged**